Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1253,6 +1253,8 @@ eval (UpdateCurrentStage stage (RideBookingRes resp)) state = do
                                 else if fareProductType == FPT.RENTAL then RENTALS 
                                 else ESTIMATES
       otp = if (( any (_ == fareProductType) [ FPT.RENTAL , FPT.INTER_CITY] ) && state.props.currentStage == RideStarted) then fromMaybe "" rideList.endOtp else if searchResultType == QUOTES then fromMaybe "" ((resp.bookingDetails)^._contents ^._otpCode) else rideList.rideOtp
+      destAddress = getAddressFromBooking (fromMaybe dummyBookingDetails (resp.bookingDetails ^._contents^.stopLocation))
```

Comment:
can we store `(fromMaybe dummyBookingDetails (resp.bookingDetails ^._contents^.stopLocation))` in a variable and use it.

Reply:
it would make it more readable and redundant code will also get removed.